Tests/CommandsTests/BuildToolTests.swift

```diff
@@ -111,11 +111,67 @@ final class BuildToolTests: XCTestCase {
         }
     }
 
+    func testNonReachableProductsAndTargetsFunctional() {
+        fixture(name: "Miscellaneous/UnreachableTargets") { path in
+            do {
+                let aPath = path.appending(component: "A")
+                let output = try execute([], packagePath: aPath)
+                XCTAssert(!output.contains("bexec"))
+                XCTAssert(!output.contains("BTarget2"))
+                XCTAssert(!output.contains("cexec"))
+                XCTAssert(!output.contains("CTarget"))
```

Comment:
probably better to check the build data than the output

Reply:
Can you elaborate?

Reply:
As in check that the expected files are present inside the .build directory